### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by Nirupma-Verma**
### Desired Outcome

*Read.md updated for explaining the usage of provider directly from registry.terraform.io and access token authentication for example for fetching secret 

### Implemented Changes

*Describe how the desired outcome above has been achieved with this PR. In
particular, consider:*

- To support access token authentication fix is being provided in conjur-api-go (https://github.com/cyberark/conjur-api-go/pull/137)
- Read.md updated with example of fetching secret using registry.terraform.io and access token

### Connected Issue/Story

Resolves #[https://github.com/cyberark/terraform-provider-conjur/issues/99
https://github.com/cyberark/terraform-provider-conjur/issues/69
]

CyberArk internal issue link: [insert issue ID]()

### Definition of Done
*At least 1 todo must be completed in the sections below for the PR to be
merged.*

#### Changelog

- [ ] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a
  CHANGELOG update

#### Test coverage

- [ ] This PR includes new unit and integration tests to go with the code
  changes, or
- [X] The changes in this PR do not require tests

#### Documentation

- [X] Docs (e.g. `README`s) were updated in this PR
- [ ] A follow-up issue to update official docs has been filed here: [insert issue ID]()
- [ ] This PR does not require updating any documentation

#### Behavior

- [ ] This PR changes product behavior and has been reviewed by a PO, or
- [ ] These changes are part of a larger initiative that will be reviewed later, or
- [X] No behavior was changed with this PR

#### Security

- [ ] Security architect has reviewed the changes in this PR,
- [ ] These changes are part of a larger initiative with a separate security review, or
- [X] There are no security aspects to these changes

### CODE REVIEW

**Comment by gcandiloro** [comment]
Not sure if my leaving a comment will help you add me as a reviewer, but it looks like I can leave feedback as a comment.

File: README.md [hunk 1]
```diff
@@ -91,6 +91,40 @@ $ mkdir -p ~/.terraform.d/plugins/
 $ # Note: If a static binary is required, use ./bin/build to create the executable
 $ go build -o ~/.terraform.d/plugins/terraform-provider-conjur main.go
 ```
+### Access from Terraform Registry
+If you wish to use the Terraform Provider Conjur from Terraform Registry
```
Comment (gcandiloro) [inline README.md:95]: Suggested change:

> If you wish to use the Terraform Provider Conjur from Terraform Registry

to

> To use the Conjur Terraform Provider from the Terraform Registry:

I would at least change Terraform Provider Conjur to Conjur Terraform Provider to match the product name.

File: README.md [hunk 2]
```diff
@@ -91,6 +91,40 @@ $ mkdir -p ~/.terraform.d/plugins/
...
+    required_providers {
+      conjur = {
+        source  = “registry.terraform.io/cyberark/conjur"
+        version = "~> 0"
+      }
+    }
+  }
+
+  provider "conjur" {
+    # All variables required variable for API Key/JWT/Access token authorization for Conjur Server. Please refer Usage section
```
Comment (gcandiloro) [inline README.md:110]: Suggestion to change:

> All variables required variable for API Key/JWT/Access token authorization for Conjur Server. Please refer Usage section

to:

> All variables required for API Key, JWT, or Access Token authentication for Conjur Server. Refer to the Usage section for details.

**Review by gcandiloro — COMMENTED** [review]
A couple of minor suggestions, all else looks good.

File: README.md (hunk 1 above)
Comment (Nirupma-Verma) [reply README.md]: Changes made and pushed

File: README.md (hunk 2 above)
Comment (Nirupma-Verma) [reply README.md]: Changes made and pushed

**Pull request closed without merging**